it/uk/gov/hmrc/soletraderidentificationfrontend/assets/TestConstants.scala

```diff
@@ -68,8 +68,15 @@ object TestConstants {
   val testSuccessfulRegistrationJson: JsObject = Json.obj(
     registrationStatusKey -> RegisteredKey,
     registeredBusinessPartnerIdKey -> testSafeId)
-  val testFailedRegistrationJson: JsObject = Json.obj(registrationStatusKey -> RegistrationFailedKey)
+  def testFailedRegistrationJson(failures: JsArray): JsObject = Json.obj(
+    registrationStatusKey -> RegistrationFailedKey,
```

Comment:
I think we should replace the keys with explicit values

<img width="1487" alt="Screenshot 2022-06-07 at 12 21 00" src="https://user-images.githubusercontent.com/4958044/172367742-12d06a75-3fc2-486a-8fb8-417eaa12eea7.png">